h2o-extensions/xgboost/src/main/java/hex/tree/xgboost/XGBoost.java

```diff
@@ -226,6 +226,10 @@
       TreeUtils.checkMonotoneConstraints(this, _train, _parms._monotone_constraints);
     }
 
+    if ((_train != null) && (_parms._tree_method == XGBoostModel.XGBoostParameters.TreeMethod.exact) || H2O.CLOUD.size() > 1) {
+      error("_tree_method", "exact is not supported in distributed environment");
```

Comment:
This should provide more details on why it is not supported (possibly a link to Technote, jira project TN)

If this is an XGBoost bug the code comment should have a link to the bug as well.

Reply:
this is not an xgboost bug, seems like a limitation of their implementation

https://github.com/dmlc/xgboost/issues/5304